lib/rocket_pants.rb

```diff
@@ -84,9 +84,12 @@ def show_exception_message=(value)
     end
 
     def default_env
-      env = Rails.env.to_s if defined?(Rails.env)
-      env ||= ENV['RAILS_ENV'].presence || ENV['RACK_ENV'].presence || "development"
-      ActiveSupport::StringInquirer.new env
+      if defined?(Rails.env) && Rails&.env
+        env = Rails.env.to_s
+      else
+        env ||= ENV['RAILS_ENV'].presence || ENV['RACK_ENV'].presence || "development"
```

Comment:
You don't really need the `||=` here. Perhaps if you want it to be clearer with the `if-else`, maybe do this instead:
```
env = if defined?(Rails.env) && Rails.env
        Rails.env.to_s
      else
        ENV["RAILS_ENV"].presence || ENV["RACK_ENV"].presence || "development"
      end
```
even though it seems to me that the original structure is fine and just add the addition `Rails.env` condition at the end of the original if.